README.md

```diff
@@ -0,0 +1,77 @@
+<img src="Logo/PNG/Header@2x.png" width="128" height="128" title="DiceKit Logo" />
+
+DiceKit
+=======
+
+[![Build Status](https://www.bitrise.io/app/bba3ae57e91a2417.svg?token=xykKCgO1PqORibgpZeeYrw&branch=master)](https://www.bitrise.io/app/bba3ae57e91a2417)
+![Supported platforms](https://img.shields.io/badge/platforms-iOS%20%2B%20OS%20X%20%2B%20watchOS-blue.svg)
+[![Latest release](https://img.shields.io/github/release/tabletopassistant/dicekit.svg)](https://github.com/TabletopAssistant/DiceKit/releases)
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg)](https://github.com/Carthage/Carthage)
```

Comment:
So, it's technically not compatible until https://github.com/Carthage/Carthage/pull/622 gets merged into a release. I think it's probably fine that I leave this here for now?